nf_core/lint/files_exist.py

```diff
@@ -81,7 +81,6 @@ def files_exist(self):
         .github/ISSUE_TEMPLATE/bug_report.md
         .github/ISSUE_TEMPLATE/feature_request.md
         docs/images/nf-core-PIPELINE_logo.png
-        .markdownlint.yml
```

Comment:
No, this is in the right place. See the context above:

> Files that *must not* be present:

If anything, we're _missing_ `yamllint.yml` here..